lib/scenic/adapters/postgres.rb

```diff
@@ -46,6 +48,16 @@ def create_view(name, sql_definition)
         execute "CREATE VIEW #{name} AS #{sql_definition};"
       end
 
+      # Updates a view in the database.
```

Comment:
Postgres does allow you to change the schema in `CREATE OR REPLACE` (it's just a drop and re-add under the covers). We opted against it because we wanted your view to have to exist first since you specifically said "update_view". Otherwise, typoing a view name for update might result in a successful migration that doesn't do what you think it was supposed to do.